src/Components/Navigation/EventNavigation.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+
+class EventNavigation extends Component {
+  constructor() {
+    super();
+    this.state = {
+      data: [],
+    };
+  }
+
+  render() {
+    return (
+      <div>
+        <EventNavBar>
+          <div className="subNaviContents">
+            <span className="subNaviTitle">액티비티</span>
+            <span className="subNaviTitle">배움</span>
+            <span className="subNaviTitle">건강 뷰티</span>
+            <span className="subNaviTitle">모임</span>
+            <div className="subNaviSubTitle">
+              {DATA.SUB_NAVI.map(element => {
+                return <span className="subNaviElement">{element}</span>;
+              })}
+            </div>
+          </div>
+        </EventNavBar>
+      </div>
+    );
+  }
+}
+
+const DATA = {
+  SUB_NAVI: [
+    "아웃도어",
+    "서핑",
+    "스포츠",
+    "수상레져",
+    "테마파크",
+    "워터파크",
+    "투어",
+    "공연",
+    "실내체험",
+  ],
+};
```

Comment:
```suggestion
const  SUB_NAV = [
    "아웃도어",
    "서핑",
    "스포츠",
    "수상레져",
    "테마파크",
    "워터파크",
    "투어",
    "공연",
    "실내체험",
  ];
```
- 굳이 객체로 한 번 더 감싸신 이유가 있을까요? 필요한 데이터 타입을 적절하게 사용해주세요 :)

Reply:
원래 페이지에는 다른 배열들이 또 들어가야해서 저렇게 구성했는데 저희가 다른페이지를 쳐내면서 저런코드가되버렸네요! 

감사합니다 수정하였습니다